tests_metricflow/snapshots/test_fill_nulls_with_0.py/str/BigQuery/test_join_to_time_spine_with_custom_grain_in_group_by__query_output.txt

```diff
@@ -0,0 +1,17 @@
+booking__ds__martian_day    bookings_join_to_time_spine
```

Comment:
This is the expected result. All of the `bookings` source data maps to this `martian_day`.